en/contributing/documentation.rst

```diff
@@ -72,26 +73,32 @@ preceded and followed by a blank line.
 Paragraphs
 ----------
 
-Paragraphs are simply blocks of text, with all the lines at the same level of indentation.
-Paragraphs should be separated by more than one empty line.
+Paragraphs are simply blocks of text, with all the lines at the same level of
+indentation. Paragraphs should be separated by more than one empty line.
 
 Inline Markup
 -------------
 
 * One asterisk: *text* for emphasis (italics)
+  We'll use it for Object names, excluding the following word "object",
+  for method/function names -- include "()".
 
-  * ``*text*``
+  * ``*PagesController*``, ``**config()**``, etc.
```

Comment:
I've been trying to use inline code for these things, but I'm flexible if the larger group thinks that italics would be better.